### DISCUSSION TITLE
Changing max token length

### DISCUSSION THREAD

**Discussion by capy-yy** [Taewan-P/gpt_mobile#130] [Q&A] [answered]
Is there a way to change the max tokens? I'm using qwq-32b-preview and since it has really long outputs having to tell it to countinue is really annoying (model: qwen/qwq-32b-preview. Provider - Hugging Face

**Comment by Taewan-P** [comment, marked as answer]
Not at the moment... not implemented. Try modifying the system prompt to get shorter answers that would work.
